### ISSUE TITLE
Gas Price in ETC is incorrectly showing 1 kWei instead of the Gwei amount

### ISSUE THREAD

**Issue by forkwatch-ninja** [hyperledger/besu#1668]
Gas prices in ETC is showing 1 Kwei.

On etcnetstats.com which has only Besu nodes running, gas prices are showing 1kwei: https://www.etcnetstats.com/

On ethercluster which is using a Besu node served publicly over RPC, it's also showing 1kwei: 

```
curl -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_gasPrice","params":[],"id":67}' https://www.ethercluster.com/etc
{
  "jsonrpc" : "2.0",
  "id" : 67,
  "result" : "0x3e8"
}
```
The hexadecimal converts to 1000.

**Comment by bobsummerwill** [comment]
By contrast ...

https://2miners.com/etc-stats/
https://expedition.dev/?network=mainnet

... show 15 Gwei.   They will both be running Core-Geth.

**Comment by bobsummerwill** [comment]
This issue was reported by MEW based on the return value they are getting from the ethercluster.com/etc endpoint, which is running Besu.

**Comment by bobsummerwill** [comment]
^ @stevanlohja, @GregTheGreek

**Comment by bobsummerwill** [comment]
Hosala: 

"it definitely started to happen recently, Ive been sampling txs on blockexplorers for etc and they all seem to have min 1gwei.
We basically tell out users to manually set the gasprice to 1 gwei and then their txs will go through, but it is annoying to respond and explain
it to lot of people"

I wonder whether this was a knock-on issue as a result of changes made for Thanos?

**Comment by shemnon** [comment]
I'm not seeing it on a local classic node we have:
```sh
$ curl -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_gasPrice","params":[],"id":67}' http://localhost:8545/
{
  "jsonrpc" : "2.0",
  "id" : 67,
  "result" : "0x24a995f60"
}
$ curl -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"admin_nodeInfo","params":[],"id":67}' http://localhost:8545/
{
  "jsonrpc" : "2.0",
  "id" : 67,
  "result" : {
    "enode" : "enode://218543c1e058d9259dac34346eefa39d73901c6a315e6f7dd54a99bcfd2f2d57867a26c0c0dabaf400e4c791e740aaa080533ecb54665b23fb9d60c9ff17e849@3.128.91.184:30303",
    "listenAddr" : "3.128.91.184:30303",
    "ip" : "3.128.91.184",
    "name" : "besu/v20.10.2/linux-x86_64/corretto-java-11",
    "id" : "218543c1e058d9259dac34346eefa39d73901c6a315e6f7dd54a99bcfd2f2d57867a26c0c0dabaf400e4c791e740aaa080533ecb54665b23fb9d60c9ff17e849",
    "ports" : {
      "discovery" : 30303,
      "listener" : 30303
    },
    "protocols" : {
      "eth" : {
        "config" : {
          "chainId" : 61,
          "homesteadBlock" : 1150000,
          "thanosBlock" : 11700000,
          "ethash" : { }
        },
        "difficulty" : 1045117316994010850929,
        "genesis" : "0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3",
        "head" : "0x26d000a5638480515521cce7ae3bbcc9ab0830d4293b90f07207dd963c1ba838",
        "network" : 1
      }
    }
  }
}
```

**Comment by shemnon** [comment]
This was fixed in 20.10.2 ( see #1546 && #1563) so upgrading ethcluster to 20.10.2 (which they should do because of the late-era thanos bug #1633)

**Comment by shemnon** [comment]
Closing since I expect ethculuser is on 20.10.1 and the fix has shipped in 20.10.2.  If not re-open.
